packages/spl/src/reward-manager/RewardManagerProgram.test.ts

```diff
@@ -138,6 +138,106 @@ describe('RewardManagerProgram', () => {
     expect(decoded.messages.length === 1)
   })
 
+  it('decodes the account data of three attestations', () => {
```

Comment:
this code goes hard. possible to provide instructions to make similar?

Reply:
like to make test cases? I got lucky and saved this from my broken rewards state on my account before claiming - you'd have to submit a few attestations without evaluating